package.json

```diff
@@ -5,32 +5,35 @@
   "repository": "https://git.studierlangsam.de/studierlangsam/webseite.git",
   "main": "gulpfile.js",
   "devDependencies": {
-    "browser-sync": "^2.26.12",
+    "browser-sync": "^2.27.5",
     "concurrent-transform": "^1.0.0",
     "emojione": "^4.5.0",
     "gulp": "^4.0.2",
     "gulp-clean": "^0.4.0",
-    "gulp-clean-css": "^4.2.0",
+    "gulp-clean-css": "^4.3.0",
     "gulp-htmlmin": "^5.0.1",
     "gulp-if": "^3.0.0",
     "gulp-image-resize": "^0.13.1",
     "gulp-plumber": "^1.2.1",
-    "gulp-pug": "^4.0.1",
+    "gulp-pug": "^5.0.0",
     "gulp-read": "1.0.1",
     "gulp-rename": "^2.0.0",
-    "gulp-rev-all": "^2.0.2",
+    "gulp-rev-all": "^3.0.0",
     "gulp-run-command": "0.0.10",
-    "gulp-sass": "^4.0.2",
+    "gulp-sass": "^5.0.0",
     "gulp-scss-lint": "^1.0.0",
-    "gulp-sourcemaps": "^2.6.5",
-    "gulp-w3cjs": "^1.3.2",
+    "gulp-sourcemaps": "^3.0.0",
+    "gulp-w3cjs": "^1.3.0",
     "rimraf": "^3.0.2",
-    "sass": "^1.26.2",
-    "yargs": "^16.0.3"
+    "sass": "^1.39.2",
+    "yargs": "^17.1.1"
   },
   "scripts": {
     "test": "gulp check"
   },
   "author": "Joshua Gleitze <dev@joshuagleitze.de>",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "npm-check-updates": "^11.8.5"
```

Comment:
This should not be a production dependency. If at all, it should be in `devDependencies`. However, I'd argue that we should not declare a dependency to `ncu` at all, but rather just have it installed on our machines.

Reply:
Actually I did install it system-wide and not in this project. Wonder how it got there. Anyways, changed that accordingly!